c2corg_ui/templates/route/helpers/detailed_route_attributes.html

```diff
@@ -51,10 +51,6 @@
       <p><span translate>configuration</span>: <b>${','.join(route['configuration'])}</b></p>
```

Comment:
``${','.join(route['configuration'])}`` will probably not be translated. Shouldn't you use something like https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/helpers/view.html#L15 ?